haas/rest.py

```diff
@@ -110,7 +110,9 @@ def rest_call(method, path, schema):
 
     * path - the url-path to map the function to. The format is the same as for
             flask's router (e.g. '/foo/<bar>/baz')
-    * method - the HTTP method for the api call (e.g. POST, GET...)
+    * method - the HTTP method for the api call (e.g. POST, GET...).
+               Can also pass a list of methods to support multiple (used in
+               testing)
```

Comment:
Wait is this just for testing then? I'd figured you were planning on using it in #554. but perhaps I've misremembered the details of that pr. If it's just for testing, I'd prefer you leave it out and try to find a way to write the tests without it.  You can stack the decorator, which may make things easier:

```python
@rest_call(...)
@rest_call(...)
def foo():
    ...
```